### ISSUE TITLE
Implement Painter's Partition Problem (Binary Search) using Java

### ISSUE THREAD

**Issue by mohdarif8299** [Algo-Phantoms/Algo-Tree#599]
@tarun26091999 @shreyanspoddar @rudrakshi99 @nakul-19 @raksha009 @OjusWiZard
Kindly allow me to implement the Painter Partition Problem using Binary Search using Java.
Kindly allow me to implement this I am good with DS & Algo using Java. I am GSSOC'21 Participant.

**Comment by mohdarif8299** [comment]
Thank you @tarun26091999. I am on it.

**Comment by rk05-lenavo** [comment]
Hey , I am participant in GSSOC'21. Can you please assign this to me? I'll solve this by using c++.

**Comment by Shraddha13Jain** [comment]
Please assign this issue to me.I can implement it in c++/c/python.I am a participant of gssoc'21

**Comment by raksha009** [comment]
@rk05-lenavo go ahead with cpp
@Shraddha13Jain go ahead with python

**Comment by rk05-lenavo** [comment]
@raksha009 I have done the code. Please check it. If it seems all right to you please merge my pull request.

**Comment by raksha009** [comment]
> @raksha009 I have done the code. Please check it. If it seems all right to you please merge my pull request.

Plz link your PR to this issue.

**Comment by Shraddha13Jain** [comment]
@raksha009 I have made the PR . please review it .

**Comment by rk05-lenavo** [comment]
@raksha009 I have made the PR. Please mearge it.

**Comment by Rishsach** [comment]
Hey, I am a participant in GSSOC'21. Can you please assign this to me? I'll solve this by using python

**Comment by tarun26091999** [comment]
> @raksha009 I have made the PR. Please mearge it.

you need to link it here, otherwise its impossible to see it.

**Comment by tarun26091999** [comment]
@rk05-lenavo link your PR here otherwise we have to unassign you from the issue.

**Comment by rk05-lenavo** [comment]
@tarun26091999 Yeah I'll link it by tomorrow night. Sorry sir for being so late. I'll definitely link it by tomorrow night.

**Comment by rk05-lenavo** [comment]
@tarun26091999 I have created the pull request. Please review the code if it seems Okay to you kindly issue me.
